### ISSUE TITLE
Unsatisfied dependencies for type MethodHandleFactory

### ISSUE THREAD

**Issue by garcia-jj** [caelum/vraptor4#184]
After #142 was merged into master, I'm getting this exception.

SEVERE: Exception sending context initialized event to listener instance of class org.jboss.weld.environment.servlet.Listener
org.jboss.weld.exceptions.DeploymentException: WELD-001408 Unsatisfied dependencies for type MethodHandleFactory with qualifiers @Default
  at injection point [BackedAnnotatedParameter] Parameter 2 of [BackedAnnotatedConstructor] @Inject public br.com.caelum.vraptor.reflection.DefaultMethodExecutor(@LRU CacheStore<Method, MethodHandle>, MethodHandleFactory)
  at br.com.caelum.vraptor.reflection.DefaultMethodExecutor.<init>(DefaultMethodExecutor.java:0)
